src/cfnlint/data/ExtendedSpecs/all/03_value_types/aws_config.json

```diff
@@ -5,14 +5,18 @@
     "value": {
       "AllowedValues": [
         "AWS::ACM::Certificate",
+        "AWS::ApiGateway::DomainName",
+        "AWS::ApiGateway::Method",
         "AWS::ApiGateway::RestApi",
         "AWS::ApiGateway::Stage",
+        "AWS::ApiGatewayV2::DomainName",
         "AWS::ApiGatewayV2::Api",
         "AWS::ApiGatewayV2::Stage",
         "AWS::AutoScaling::AutoScalingGroup",
         "AWS::AutoScaling::LaunchConfiguration",
         "AWS::AutoScaling::ScalingPolicy",
         "AWS::AutoScaling::ScheduledAction",
+        "AWS::CertificateManager::Certificate",
```

Comment:
Do you know if this one should be [`AWS::ACM::Certificate`](https://docs.aws.amazon.com/config/latest/developerguide/resource-config-reference.html#awscertificatemanager) or [`AWS::CertificateManager::Certificate`](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-certificatemanager-certificate.html)?

Reply:
This one has also disappeared in the last week as an option. I think it was replaced by AWS::ACM::Certificate.